src/widgets/Button.js

```diff
@@ -17,6 +17,28 @@ export default class Button extends createjs.Container {
       displayObject: this,
       accessibleOptions: options,
       role: AccessibilityModule.ROLES.BUTTON,
+      events: [
```

Comment:
I like this better than what I was thinking of for event registration!  And it has the added flexibility of allowing for binding multiple listeners for the same event.